docs-user/guide-remote-profiling.md

```diff
@@ -1,28 +1,125 @@
 
 ![A screenshot of the new performance panel](./images/new-performance-panel.png)
 
-# Tips
+## Symbols and symbol sources
+
+If you've been profiling one of the builds suggested in this document, your profile should contain fully symbolicated C++ call stacks at least for libxul.so. If not, it probably was not a "Nightly" GeckoView build; regular GeckoView build jobs  on treeherder don't upload symbols to the Mozilla symbol server.
+
+For Android system libraries, symbol information can be a bit hit and miss. Depending on your Android version, if the browser on the phone is running Gecko 64 or newer, sometimes you will get symbols for system libraries, sometimes you won't; if you don't, the patch in [bug 1505719](https://bugzilla.mozilla.org/show_bug.cgi?id=1505719) should help. It's currently in review.
+
+There are other types of builds you might want to profile:
+
+### Try builds
+
+If you want to profile an Android build that the tryserver created for you, you have to kick off a "Sym" job on treeherder: Using treeherder's *Add new jobs* UI, schedule a "Sym" job for each platform whose "B" job you want symbols for. (And "SymN" if you have an "N" job you want symbols for.) These jobs gather symbol information from the corresponding build job and upload it to the Mozilla symbol server so that perf.html can use it.
+
+### Local builds
+
+If you've compiled an Android Gecko build locally, and want to profile it, you have to jump through a few extra hoops at the moment unfortunately.
+
+At the moment, Firefox Nightly Desktop doesn't support symbolicating profiles that were obtained from local Android Gecko builds. However, [patches are](https://bugzilla.mozilla.org/show_bug.cgi?id=1509549) [in review](https://bugzilla.mozilla.org/show_bug.cgi?id=1504101) to enable just that, and you can try out builds that include those patches: From [this try push](https://treeherder.mozilla.org/#/jobs?repo=try&revision=6b2b61d055279cb5dff4361fc428460891d6ddd2), download the Firefox Desktop build for your host platform, and connect it to your custom Android build through WebIDE. Before profiling, in the *Performance* panel in WebIDE, open the *Local build* section and add your Android build's objdir to the list. Then profile as usual, and you should be getting full symbol information.
+
+## Tips
 
 * Avoid opening any other panels besides the Performance panel, to reduce the overhead of activated panels.
 * After connecting to the device, do not select *Main Process* or any of the open tabs from the list of targets on the left.
-* If WebIDE automatically reconnects to a debugging target from a previous session, disconnect, uncheck the two checkboxes in the WebIDE preferences (`Project -> Preferences`) and reconnect to the device.
+* If WebIDE automatically reconnects to a debugging target from a previous session, disconnect, uncheck the two checkboxes in the WebIDE preferences (*Project* > *Preferences*) and reconnect to the device.
```

Comment:
There's a tip we can add (here or in the "recording" session): if the recording doesn't start, it might be necessary to close and restart WebIDE to reset some internal state. (we really want to either fix this or help with the about:debugging effort to replace WebIDE).